src/components/Footer/Contacts.tsx

```diff
@@ -14,12 +14,12 @@ export const Contacts = JSX<ContactsProps>(({ className, items, hasButton, conte
   const { handlerDecorator } = context;
 
   return (
-    <div className={`flex flex-col ${className || ''}`}>
+    <div className={`${className || ''}`}>
```

Comment:
```suggestion
    <div className={className}>
```

А там где деструктуризация пропсов присвоить значение по умолчанию - пустая строка.